lib/identity_doc_auth/errors.rb

```diff
@@ -7,7 +7,8 @@ module Errors
     CONTROL_NUMBER_CHECK = 'control_number_check'
     DOC_CROSSCHECK = 'doc_crosscheck'
     DOC_NUMBER_CHECKS = 'doc_number_checks'
-    EXPIRATION_CHECKS = 'expiration_checks'
+    DOCUMENT_EXPIRED = 'doc_expired' # document has expired
```

Comment:
```suggestion
    DOCUMENT_EXPIRED = 'doc_expired_check' # document has expired
```

consistency again

Reply:
Did you mean to rename the constant here too?

Reply:
oops, yep